Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -0,0 +1,220 @@
+
+                              zoomLevel: 13, animated: false)
+
+            // Setup offline pack notification handlers.
+            NotificationCenter.default.addObserver(self, selector: #selector(offlinePackProgressDidChange), name: NSNotification.Name.MGLOfflinePackProgressChanged, object: nil)
+            NotificationCenter.default.addObserver(self, selector: #selector(offlinePackDidReceiveError), name: NSNotification.Name.MGLOfflinePackError, object: nil)
+            NotificationCenter.default.addObserver(self, selector: #selector(offlinePackDidReceiveMaximumAllowedMapboxTiles), name: NSNotification.Name.MGLOfflinePackMaximumMapboxTilesReached, object: nil)
+            installConstraints()
+        }
+
+        func installConstraints() {
+
+            if #available(iOS 11.0, *) {
+                NSLayoutConstraint.activate([
+                    downloadButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+                    downloadButton.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -170),
+                    downloadButton.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor)
+                ])
+            } else {
+                // Fallback on earlier versions
+            }
+
+        }
+
+        override func viewDidDisappear(_ animated: Bool) {
+
+            super.viewDidDisappear(animated)
+            // When leaving this view controller, suspend offline downloads.
+            guard let packs = MGLOfflineStorage.shared.packs else { return }
+            for pack in packs {
+                if let userInfo = NSKeyedUnarchiver.unarchiveObject(with: pack.context) as? [String: String] {
+                    print("Suspending download of offline pack: “\(userInfo["name"] ?? "unknown")”")
+                }
+                pack.suspend()
+            }
+        }
+
+        deinit {
+            // Remove offline pack observers.
+            print("Removing offline pack notification observers")
+            NotificationCenter.default.removeObserver(self)
+
+        }
+
+        func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+            if packArray.isEmpty  {
+                return 1
+            } else {
+                return packArray.count
+            }
+
+        }
+
+
+        func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+            let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+            if MGLOfflineStorage.shared.packs?.count != nil {
+                let progress = Float(packArray[indexPath.row].progress.countOfResourcesCompleted)/Float(packArray[indexPath.row].progress.countOfResourcesExpected)
+                cell.textLabel!.text = "Region \(indexPath.row + 1): \n  Percent completion: \(String(format: "%.2f", progress * 100)) %"
```

Comment:
I'd recommend against having a line break here - maybe just a dash instead? Then we don't need to explicitly set the cell height.